tests/js/server/replication2/replication2-supervision-replicated-log-target-cluster.js

```diff
@@ -48,7 +49,7 @@ const waitForReplicatedLogAvailable = function (id) {
     while (true) {
         try {
             let status = db._replicatedLog(id).status();
-            if (status.role === "leader") {
+            if ("logStatus" in status && status.logStatus.role === "leader") {
```

Comment:
This looks like python to me, but it is [actually](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Operators/in) valid js 😆